README.md

```diff
@@ -54,73 +89,67 @@ The instance of an Optimization problem is the valid input for instantiating the
 
 ## Tutorials
 
-### QP Tutorial
+### Quadratic Programming
 - [Solving LASSO.](https://github.com/lava-nc/lava-optimization/blob/release/v0.2.0/tutorials/tutorial_01_solving_lasso.ipynb)
 
-### Solving QP problems (After merging with the OptimizationSolver QPSolver will be deprecated)
```

Comment:
I like that it looks cleaner without the explanation of QPSolver vs OptimizationSolver here and in the QUBO example, but then we should add a section at the end with this clarification.